src/utils/permissions.js

```diff
@@ -2,7 +2,7 @@
 import Selectors from '../selectors'
 import type { ClusterType, PermissionType } from '../ovirtapi/types'
 
-function checkPermissions (allowedPermissions: Array<string>, permissions: Array<PermissionType>): boolean {
+function checkPermissions (permission: string, permissions: Array<PermissionType>): boolean {
```

Comment:
No, because purpose of `checkPermissions` is to check that user have permissions to do something. Previous it was array, because roles can contain same permits, but now we don't need to check bunch of permissions, we can check permit directly.